statistics/table.go

```diff
@@ -473,16 +470,17 @@ func isSingleColIdxNullRange(idx *Index, ran *ranger.Range) bool {
 // It assumes all modifications are insertions and all new-inserted rows are uniformly distributed
 // and has the same distribution with analyzed rows, which means each unique value should have the
 // same number of rows(Tot/NDV) of it.
-func outOfRangeEQSelectivity(ndv, modifyRows, totalRows int64) float64 {
-	if modifyRows == 0 {
+func outOfRangeEQSelectivity(ndv, tableRowCount, totalRows int64) float64 {
```

Comment:
`tableRowCount` and `totalRows` seems similar and confused.
How about using `currentRowCount` and `statsRowCount`?